include/aws/common/assert.h

```diff
@@ -45,8 +45,10 @@ AWS_EXTERN_C_END
             bool _result = (cond);                                                                                     \
             __builtin_assume(_result);                                                                                 \
         } while (false)
-#elif defined(__GNUC__)
+#elif defined(__GNUC__) && __GNUC__ >= 4 && __GNUC_MINOR__ >= 5
```

Comment:
uuuuuh so gcc 8.1 for example doesn't get this branch?

Reply:
Well crap! I’ll have to do another fix.